File: codeforces/11.stones_on_the_table/sott.py
# Author: José Rodolfo (jric2002)
n = int(input())
stones_taken = 0
if (n != 1):
  row_of_stones = str(input())
  for position in range(0, (len(row_of_stones) - 1)):
    if (row_of_stones[position] == row_of_stones[position + 1]):
      stones_taken += 1
print(stones_taken)

File: codeforces/11.stones_on_the_table_[266A]/sott.cpp
#include <iostream>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned short int n;
  unsigned short int stones_taken = 0;
  char current_stone, previous_stone;
  cin >> n;
  if (n != 1) {
    cin >> previous_stone;
    for (unsigned short int i = 0; i < (n - 1); i++) {
      cin >> current_stone;
      if (current_stone == previous_stone) {
        stones_taken++;
      }
      previous_stone = current_stone;
    }
  }
  cout << stones_taken << endl;
  return 0;
}